### ISSUE TITLE
[NV-820] - Error in logs: (0 , hub_1.withScope) is not a function

### ISSUE THREAD

**Issue by novubot** [novuhq/novu#1105]
## What?

Unfortuently couldn't find the exact reason when it happens or how to replicate this, the log below is the only thing we have available:

```
2022-08-22 04:48:26TypeError: (0 , hub_1.withScope) is not a function4c52e837006d453cbfd75cc78983ce13
2022-08-22 04:48:26at Object.error (/usr/src/app/node_modules/.pnpm/nest-raven@8.2.0_mr44slrmkv27po44cqxkckehi4/node_modules/nest-raven/dist/raven.interceptor.js:34:41)4c52e837006d453cbfd75cc78983ce13
2022-08-22 04:48:26at /usr/src/app/node_modules/.pnpm/rxjs@7.5.5/node_modules/rxjs/dist/cjs/internal/operators/tap.js:30:824c52e837006d453cbfd75cc78983ce13
2022-08-22 04:48:26at OperatorSubscriber._this._error (/usr/src/app/node_modules/.pnpm/rxjs@7.5.5/node_modules/rxjs/dist/cjs/internal/operators/OperatorSubscriber.js:43:21)4c52e837006d453cbfd75cc78983ce13
2022-08-22 04:48:26at OperatorSubscriber.Subscriber.error (/usr/src/app/node_modules/.pnpm/rxjs@7.5.5/node_modules/rxjs/dist/cjs/internal/Subscriber.js:60:18)4c52e837006d453cbfd75cc78983ce13
2022-08-22 04:48:26at OperatorSubscriber.Subscriber._error (/usr/src/app/node_modules/.pnpm/rxjs@7.5.5/node_modules/rxjs/dist/cjs/internal/Subscriber.js:84:30)4c52e837006d453cbfd75cc78983ce13
2022-08-22 04:48:26at OperatorSubscriber.Subscriber.error (/usr/src/app/node_modules/.pnpm/rxjs@7.5.5/node_modules/rxjs/dist/cjs/internal/Subscriber.js:60:18)4c52e837006d453cbfd75cc78983ce13
2022-08-22 04:48:26at OperatorSubscriber.Subscriber._error (/usr/src/app/node_modules/.pnpm/rxjs@7.5.5/node_modules/rxjs/dist/cjs/internal/Subscriber.js:84:30)4c52e837006d453cbfd75cc78983ce13
2022-08-22 04:48:26at OperatorSubscriber.Subscriber.error (/usr/src/app/node_modules/.pnpm/rxjs@7.5.5/node_modules/rxjs/dist/cjs/internal/Subscriber.js:60:18)4c52e837006d453cbfd75cc78983ce13
2022-08-22 04:48:26at OperatorSubscriber.Subscriber._error (/usr/src/app/node_modules/.pnpm/rxjs@7.5.5/node_modules/rxjs/dist/cjs/internal/Subscriber.js:84:30)4c52e837006d453cbfd75cc78983ce13
2022-08-22 04:48:26at OperatorSubscriber.Subscriber.error (/usr/src/app/node_modules/.pnpm/rxjs@7.5.5/node_modules/rxjs/dist/cjs/internal/Subscriber.js:60:18)4c52e837006d453cbfd75cc78983ce13
2022-08-22 04:48:26at OperatorSubscriber.Subscriber._error (/usr/src/app/node_modules/.pnpm/rxjs@7.5.5/node_modules/rxjs/dist/cjs/internal/Subscriber.js:84:30)4c52e837006d453cbfd75cc78983ce13
2022-08-22 04:48:26at OperatorSubscriber.Subscriber.error (/usr/src/app/node_modules/.pnpm/rxjs@7.5.5/node_modules/rxjs/dist/cjs/internal/Subscriber.js:60:18)4c52e837006d453cbfd75cc78983ce13
2022-08-22 04:48:26at OperatorSubscriber.Subscriber._error (/usr/src/app/node_modules/.pnpm/rxjs@7.5.5/node_modules/rxjs/dist/cjs/internal/Subscriber.js:84:30)4c52e837006d453cbfd75cc78983ce13
2022-08-22 04:48:26at OperatorSubscriber.Subscriber.error (/usr/src/app/node_modules/.pnpm/rxjs@7.5.5/node_modules/rxjs/dist/cjs/internal/Subscriber.js:60:18)4c52e837006d453cbfd75cc78983ce13
2022-08-22 04:48:26at OperatorSubscriber.Subscriber._error (/usr/src/app/node_modules/.pnpm/rxjs@7.5.5/node_modules/rxjs/dist/cjs/internal/Subscriber.js:84:30)4c52e837006d453cbfd75cc78983ce13
```

**Comment by ianchagas** [comment]
Looking for a possible solution to the problem, I found this issue. I have the same problem (maybe in another context) but I don't know what to do.
Localhost running on Docker works, but on the server, and in only one specific route of my application this error happens.

If anyone can help, I appreciate it!

![unknown](https://user-images.githubusercontent.com/71187950/187470570-36742a6f-917b-403c-add4-f68dbc294733.png)

**Comment by scopsy** [comment]
Wonder if thats a bug from @nestjs/sentry.
Will try to upgrade it maybe will help :|

**Comment by ianchagas** [comment]
@scopsy 

I removed everything related to Sentry from the application, and after that, it pointed out in the server log that what happened was a TypeORM error (which I also use here) that is using some deprecated methods when deleting information in the route, thus generating a silent error.
That is, it could be that the error is not directly related to Sentry, but rather related to a repository error or service error.

![image](https://user-images.githubusercontent.com/71187950/187773749-323127cb-3f11-467c-8d3c-26752e9d9924.png)

Now it's easier to solve here, but here's an idea of a solution for you

**Comment by scopsy** [comment]
Thanks, @ianchagas it's definitely something we will try to look at it. Will update if found something

**Comment by scopsy** [comment]
@ianchagas managed to fix this at: #1218 by upgrading the @sentr/node package. It was hiding the real exception behind this crypting message before. After the upgrade I don't see these errors any more.
